RELEASE_NOTES.md

```diff
@@ -9,27 +9,24 @@
 [//]: # ( - Configurations public constructor are deprecated, please use each Configuration's builder to make a Configuration object)
 
 ## New
-- Added support for 6 more locales: Catalan (ca-ES), Icelandic (is-IS), Bulgarian (bg-BG),
-  Estonian (et-EE), Latvian (lv-LV) and Lithuanian (lt-lT).
+- For Twint, storing payment details and paying with them is now supported. See the documentation [here](/docs/payment-methods/TWINT.md).
 - French meal vouchers are now available with the following payment method types. See the documentation [here](/docs/payment-methods/FRENCH_MEAL_VOUCHER.md).
   - Up. Payment method type: **mealVoucher_FR_groupeup**.
   - Natixis. Payment method type: **mealVoucher_FR_natixis**.
   - Sodexo. Payment method type: **mealVoucher_FR_sodexo**.
 - For API only merchants, `CardNumberValidator`, `CardExpiryDateValidator` and `CardSecurityCodeValidator` classes are added to make validation functionality for the corresponding fields available to public.
-- For Twint, storing payment details and paying with them is now supported. See the documentation [here](/docs/payment-methods/TWINT.md).
 - You can now use [Adyen Test Cards Android](https://github.com/Adyen/adyen-testcards-android) to prefill test payment method information, making testing your integration easier.
+- Added support for 6 more locales: Catalan (ca-ES), Icelandic (is-IS), Bulgarian (bg-BG), Estonian (et-EE), Latvian (lv-LV) and Lithuanian (lt-lT).
 
 > [!WARNING]
 > For the Twint component integration, you are now required to use `TwintComponent` instead of `InstantPaymentComponent`. See the [documentation](/docs/payment-methods/TWINT.md) to find out the details.
 
 ## Improved
-- For UPI Intent an error message will be shown when "Continue" button is pressed without selecting
-  any UPI option.
+- For UPI Intent an error message will be shown when "Continue" button is pressed without selecting any UPI option.
 - For drop-in, improved accessibility of back/close button in the navigation bar.
 
 ## Changed
-- For drop-in, headers of preselected stored payment screen and payment methods list screen are
-  updated.
+- For drop-in, headers of preselected stored payment screen and payment methods list screen are updated.
 - Dependency versions:
```

Comment:
This table needs to be updated, the release guide explains how to get all the modified deps